pt-BR/mail/chrome/messenger/preferences/general.dtd

```diff
@@ -1,10 +1,10 @@
-<!ENTITY alwaysCheckDefault.label          "Always check to see if &brandShortName; is the default mail client on startup">
+<!ENTITY alwaysCheckDefault.label          "Sempre verificar se o &brandShortName; é o cliente padrão de e-mails ao iniciar">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY alwaysCheckDefault.label          "Sempre verificar se o &brandShortName; é o cliente padrão de correio eletrônico ao iniciar">